test/proxy.conf.template

```diff
@@ -16,18 +16,18 @@ server {
     proxy_set_header   X-Forwarded-Host $server_name;
 
     location ~ ^/items/.*/revisions.* {
-        proxy_pass         ${NGINX_REROUTE_HOST};
+        proxy_pass http://syncing-server-js;
     }
 
     location ~ ^/session.* {
-        proxy_pass         ${NGINX_REROUTE_HOST};
+        proxy_pass http://syncing-server-js;
     }
 
     location ~ ^/auth.* {
-        proxy_pass         ${NGINX_REROUTE_HOST};
+        proxy_pass http://syncing-server-js;
```

Comment:
so `auth` locations get proxied to `syncing-server-js`: is this intended?

Reply:
so this is Load Balancer config ... which is routing traffic only between syncing-server and syncing-server-js

whereas `auth` is standing behind the API Gateway ... so only urls like `.../v1/..` go to API Gateway and land on Auth